app/controllers/user_controller.rb

```diff
@@ -338,7 +338,8 @@ def contact
           params[:contact][:subject],
           params[:contact][:message]
         ).deliver
-        flash[:notice] = _("Your message to {{recipient_user_name}} has been sent!",:recipient_user_name=>CGI.escapeHTML(@recipient_user.name))
+        flash[:notice] = _("Your message to {{recipient_user_name}} has been sent!",
+                           :recipient_user_name => @recipient_user.name.html_safe)
```

Comment:
if the name contains markup that's not marked as html safe, it gets escaped during translation then escaped by JSON when the cookie is read